config/weekly_checks.yaml

```diff
@@ -0,0 +1,36 @@
+version: 0.0.1
+name: 'Weekly Checks'
+
+input:
+  files:
+    - data/L1Ntuple_*.root
+  sample:
+    name: Data
+    title: 2016 Data
+  trigger:
+    name: SingleMu
+    title: Single Muon
+  pileup_file: ""
+  run_number: 276243
```

Comment:
I think it's worth putting something obviously junk-like in here so we know the person using this config should update it, and it's obvious if they forget.

Reply:
I Agree.
The initial set of config parameters is taken from Shane's macros and needs tuning.
I want to use the weekly config 'exercise' to find these things and eliminate them bit-by-bit.